pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -223,6 +224,7 @@ public PinotConfiguration getConfig() {
   public void start()
       throws Exception {
     LOGGER.info("Starting Pinot broker (Version: {})", PinotVersion.VERSION);
+    LOGGER.info(new PinotAppConfigs(getConfig()).getJvmInputArguments());
```

Comment:
We can probably log everything within the `PinotAppConfigs` by calling:
```suggestion
    LOGGER.info("Broker configs: {}", new PinotAppConfigs(getConfig()).toJSONString());
```

Same for other components

Reply:
Thanks for the review. Let me make the changes.